dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/services/jobService/stage/PrepStage.kt

```diff
@@ -0,0 +1,23 @@
+package com.spectralogic.dsbrowser.gui.services.jobService.stage
+
+import com.google.common.collect.ImmutableMap
+import com.spectralogic.ds3client.helpers.Ds3ClientHelpers
+import java.nio.file.Path
+
+/*
+ * ****************************************************************************
+ *    Copyright 2014-2017 Spectra Logic Corporation. All Rights Reserved.
+ *    Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *    this file except in compliance with the License. A copy of the License is located at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    or in the "license" file accompanying this file.
+ *    This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *    CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *    specific language governing permissions and limitations under the License.
+ *  ****************************************************************************
+ */
```

Comment:
move the header to be the first thing in the file

Reply:
?

Reply:
I see it as the first thing in the file, GitHub does not seem to.